src/generator/predefined/container.js

```diff
@@ -34,4 +37,42 @@ export class __$NAME__ extends Component {
 export default connect(
   (/* state */) => ({/** _INSERT_STATE_  **/}),
   (dispatch) => bindActionCreators({/** _INSERT_ACTION_CREATORS_ **/}, dispatch)
-)(__$NAME__);
+)(${args => args.name});
+`;
+
+const testTemplate = createTemplate`
+/*global React*/
+/*global describe it*/
+/*global expect*/
+import { ${args => args.name} } from "${args => args.path}";
+import { shallow } from "enzyme";
+
+describe("${args => args.name}", () => {
+  it("renders without an issue", () => {
+    const subject = <${args => args.name} />;
+    const wrapper = shallow(subject);
+    expect(wrapper).to.exist;
+  });
+});
+`;
+
+module.exports = exports = options => ({
+  args: {
+    name: options.name
+  },
+  entries: [
+    {
+      path: "src/containers",
+      filename: options.name,
+      template: containerTemplate
+    },
+    {
+      path: "test/containers",
+      filename: `${options.name}.test.js`,
+      template: testTemplate,
+      args: {
+        path: `../../src/containers/${options.name}`
```

Comment:
containers also has an alias so we should be able to do
`import ContainerName from "containers/ContainerName";`